test/UnitSPIRV.h

```diff
@@ -99,6 +99,7 @@ class WordVector {
 
 inline void PrintTo(const WordVector& words, ::std::ostream* os) {
   size_t count = 0;
+  const auto saved_flags = os->flags();
```

Comment:
The width is reset after operator<< on "value".
(See Notes in http://en.cppreference.com/w/cpp/io/manip/setw)
However, you're right that std::hex isn't, so it's useful to save and restore the flags.

However, my reading of the reference pages says the fill character is not reset automatically. So we have to save and restore that too.

But these are preexisting problems with the code, so this CL is a step forward.